riak_test/opbcounter_test.erl

```diff
@@ -33,7 +37,7 @@ confirm() ->
     decrement_test(Nodes),
     transfer_test(Nodes),
     conditional_write_test(Nodes),
-    rt:clean_cluster(Nodes),
+    %rt:clean_cluster(Nodes),
```

Comment:
To all test or only to the test I removed it?

> On 14 Sep 2015, at 14:06, Annette Bieniusa <notifications@github.com> wrote:
> 
> In riak_test/opbcounter_test.erl <https://github.com/SyncFree/antidote/pull/143#discussion_r39390610>:
> 
> > @@ -33,7 +37,7 @@ confirm() ->
> >      decrement_test(Nodes),
> >      transfer_test(Nodes),
> >      conditional_write_test(Nodes),
> > -    rt:clean_cluster(Nodes),
> > +    %rt:clean_cluster(Nodes),
> This is what I said: Using different keys will break, because programmers are not careful. 
> Please re-add the clean up.
> 
> Annette
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/SyncFree/antidote/pull/143/files#r39390610>.
>